CHANGELOG.md

```diff
@@ -1,7 +1,288 @@
+  - Re-enabled `g++-9` + CUDA jobs #2040
+  - Updated Read the Docs configuration to v2 #2010
+  - For ROCm versions <= 5.3 certain warnings are ignored #1932
+  - Split compile and runtime CI runners into separate GitLab pipelines #1908
+  - Switched more CI runners to C++20 mode #1902
+  - LLVM sanitizer libraries are explicitly installed #1900
+  - Re-enabled CUDA + `gcc-10` jobs #1890
+  - Moved all GitHub jobs from `ubuntu-latest` to `ubuntu-20.04` #1872
+  - More jobs are only compiling the test cases but no longer execute them #1869
+  - CUDA CI runners no longer manually install the GPU driver #1853
+  - Change ROCm CI node #1844
+  - Reworked Xcode OpenMP installation #1840 #1922
+  - Upgraded to GitHub checkout action v3 #1832
+  - Upgraded to `clang-format-14` #1831
+  - Upgraded test infrastructure to Catch2 v3 #1749 #1815 #1861 #1911 
+  - Upgraded headercheck CI run to clang-13 and CUDA 11.2 #1803
+  - Simplified CI clang installation #1763
+  - Running CI workflows are now automatically cancelled when their corresponding PRs are updated #1717
+
+### Deprecated
+
+- **Breaking change**: deprecated `alpaka::getPitchBytes[Vec]` functions in favour of new `alpaka::getPitchesInBytes` function #2092 #2116
+- **Breaking change**: deprecated `alpaka::get{Extent,Offset}[Vec]` functions in favour of new `alpaka::get{Extents,Offsets}` functions #2080 #2139
+
+### Removed
+
+- `g++`:
+  - Dropped support for `g++-{7,8}` #1872
+- `clang++`:
+  - Removed work-around for very old clang versions #1916
+  - Dropped support for clang as CUDA compiler for all versions before `clang-14` #1890
+  - Dropped support for `clang-{6,7,8,9}` #1872
+  - Dropped support for `clang-5` #1750
+- `icpc`:
+  - Dropped support for the Intel® C++ Compiler Classic (`icpc`) #1702
+- MSVC:
+  - Temporarily dropped support for MSVC + CUDA due to a nvcc bug #1958
+  - Dropped support for MSVC 2019 #1887
+- Xcode:
+  - Dropped support for Xcode 12.4.0 #1759
+- CUDA:
+  - Dropped support for CUDA 10 #1872
+  - Dropped support for CUDA 9.2 #1855
+- ROCm:
+  - Dropped support for ROCm 4 #1886
+- SYCL:
+  - Removed Xilinx platform support #1970
+- Removed floating point contractions for math test cases #2155
+- Removed `alpaka::set{Extent,Offset}` functions #2087
+- Removed alpaka's experimental accessors #2054 #2062
+- Catch2 is no longer compiled with `CATCH_CONFIG_FAST_COMPILE` set to `ON` #1978
+- Removed OpenMP 5 back-end #1947
+- Removed OpenACC back-end #1941
+- Removed warning for Boost 1.73 since alpaka requires Boost >= 1.74 #1849
+- Removed previously deprecated `alpaka::time` functionality #1841
+- Removed `alpaka::{map,unmap,pin,unpin,isPinned,prepareForAsyncCopy}()` free functions #1790
+- Removed unused `alpaka::ConceptUniformCudaHip` #1736
+- Removed Boost.fiber back-end #1718
+
+### Fixed
+
+- **Potentially breaking change**: Work around a `g++-11` bug. This requires an additional data member inside `alpaka::PlatformCpu` #2165
```

Comment:
Since platforms have not been objects before, this is not a breaking change wrt. to alpaka 0.9.